appendix-11-asking for help.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: page
+title: "Asking for help"
+chapter: "Chapter A10:"
+prev_link: "← Holidays"
+prev_url: "/appendix-09-holidays"
+---
+
+As a fully distributed team, remote calls our lifeblood for our communication. Simple joining a remote team doesn't make you great at communicating remotely, though, and at some point, we're all going to need help. So who do we turn to? How do we ask? Here are a few essential tips we've developed to successfully ask for help within a remote team.
+
+## Be willing to ask
+We all need help, and for it's inherently human to feel embarrassed to ask for it. This is something as a team we strive to push through. Asking for help is strength, not weakness. Asking for help grows you, and it empowers the individual who knows the answer to practice sharing that information. This is a balance, of course. There is such a thing as too many questions in a given period. Do your research, If it's a work question, check the handbook. A code question, stack overflow. Use your discretion, and when all else fails, apply the 5-minute rule and phone a friend.
```

Comment:
> We all need help, and for it's inherently human

We all need help, and ~~for~~ it's inherently human

> Do your research, If it's a work

I think maybe that should be a period.

Reply:
These are both addressed!